omod/src/main/webapp/resources/jsonforms/HMIS-LAB-001-GENERAL-LABORATORY-TEST-REQUEST-FORM.json

```diff
@@ -49,7 +49,6 @@
             {
```

Comment:
Keep name consistent to be same as old from https://github.com/METS-Programme/openmrs-module-ugandaemr/blob/6db31049fa3b82eb6b474e3121890cae04c93de6/omod/src/main/webapp/resources/jsonforms/HMIS-LAB-001-GENERAL-LABORATORY-TEST-REQUEST-FORM.json#L2

Reply:
Ensure encounter type matches that of html form https://github.com/METS-Programme/openmrs-module-ugandaemr/blob/6db31049fa3b82eb6b474e3121890cae04c93de6/omod/src/main/webapp/resources/jsonforms/HMIS-LAB-001-GENERAL-LABORATORY-TEST-REQUEST-FORM.json#L5C21-L5C57

"cbf01392-ca29-11e9-a32f-2a2ae2dbcce4"